napari/layers/labels/_tests/test_labels.py

```diff
@@ -1562,6 +1562,23 @@ def test_color_mapping_when_seed_is_changed():
     assert not np.allclose(mapped_colors1, mapped_colors2)
 
 
+@pytest.mark.parametrize(
+    'num_colors', [49, 50, 254, 255, 60000, 65534, 2**17]
```

Comment:
```suggestion
    'num_colors', [49, 50, 254, 255, 60000, 65534, 2**17]
```
But we do not support over 2**16 

https://github.com/napari/napari/blob/6524ee417274c2feaa24d504c0ddc4e57c2fd355/napari/layers/labels/labels.py#L564-L565

Reply:
Well, the tests pass 😅 So I guess we don't have this check either in the constructor or when changing the colormap — which is the future of the function anyway.

Reply:
Yes. There is no check for this in the constructor. I think that we should fix this (maybe separate PR). But rendering will fail for `num_colors` above 2**16

Reply:
Can we address this in a later PR? Or do you want me to remove the check? (I don't see a good reason to fail in this specific function — maybe in the future we allow large colormap rendering / float32 textures?)

Reply:
I think that we could add check in followup PR, But I prefer to remove this case from tests. 

I prefer to raise an exception on LabelColormap create not, to execute vispy code as it will be easier to understand by user.

Reply:
Ok I've removed the test case.